TransferMethod/Sources/AddTransferMethodPresenter.swift

```diff
@@ -206,6 +206,12 @@ final class AddTransferMethodPresenter {
                                                    transferMethodProfileType: profileType)
                     .build()
 
+        case HyperwalletTransferMethod.TransferMethodType.paperCheck.rawValue:
+            return HyperwalletVenmoAccount.Builder(transferMethodCountry: country,
```

Comment:
This has to be HyperwalletPaperCheckAccount instead of HyperwalletVenmoAccount. Once core sdk pr is ready you can update this.

Reply:
ok